### ISSUE TITLE
New rule: no-multiline-expressions

### ISSUE THREAD

**Issue by 43081j** [43081j/eslint-plugin-lit#138]
Not sure of the best name for this but some of us prefer that complex expressions are moved out into a variable rather than being inlined.

In that case, this would be incorrect:

```ts
html`
  ${xyz
    .foo()
    .bar()}
`;
```

of course this shouldn't be added to the recommended config since it is _very_ opinionated.

cc @stramel for opinions maybe

**Comment by 43081j** [comment]
this doesn't really belong in a lit-specific plugin im suspecting.

i have no clue where it _does_ belong though 🙈

**Comment by 43081j** [comment]
i'm gonna close this as i don't believe it belongs in a lit specific plugin after having spent some time thinking about it.
